notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -825,9 +791,9 @@
         "id": "tarball_training_script"
       },
       "source": [
-        "**Store the training script on your Cloud Storage bucket.**\n",
+        "### Store the training script in your Cloud Storage bucket.\n",
         "\n",
-        "Next, you package the training folder into a compressed tar ball, and then store it in your Cloud Storage bucket."
+        "Next, package the training folder into a compressed tar ball, and then store it in your Cloud Storage bucket."
```

Comment:
change "it" to "the folder":

- "Next, package the training folder into a compressed tar ball, and then store the folder in your Cloud Storage bucket."